src/build/build_helpers.jl

```diff
@@ -122,8 +129,7 @@ function get_jump_variable_for_commodity(jm, commodity::IndexedCommodity)
     return jm[get_name(commodity)][commodity.subindex]
 end
 
-function get_jump_expression_for_commodity_producer_price(m::Model, jm, pf, commodity::CommodityRef)
-    jump_commodity = get_jump_variable_for_commodity(jm, commodity)
+function get_expression_for_commodity_producer_price(pf, commodity::CommodityRef)
```

Comment:
Here and all similar in this branch I updated the function name, taking out "jump", and 2 redundant arguments (jm, and m)